### PR TITLE
Add pdf2svg

### PR DESCRIPTION
**Pull request by yujincheng08**

### CODE REVIEW

File: mingw-w64-pdf2svg/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Maintainer: Jaroslav Lichtblau <svetlemodry@archlinux.org>
+# Contributor: Stefan Husmann <stefan-husmann@t-online.de>
+# Contributor: corvolino <corvolino@archlinux.com.br>
+# Contributor: Joel Schaerer
+# Contributor: LoveSy
```
Comment (filnet) [inline mingw-w64-pdf2svg/PKGBUILD]: You should put yourself as Maintainer and remove the Contributors.

File: mingw-w64-pdf2svg/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+_realname=pdf2svg
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=0.2.3
+pkgrel=2
```
Comment (filnet) [inline mingw-w64-pdf2svg/PKGBUILD]: pkgrel should be 1.

File: mingw-w64-pdf2svg/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+pkgdesc="A pdf to svg converter"
```
Comment (filnet) [inline mingw-w64-pdf2svg/PKGBUILD]: Add (mingw-w64) at the end of the description : pkgdesc="A pdf to svg converter (mingw-w64)"

File: mingw-w64-pdf2svg/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=0.2.3
+pkgrel=1
+pkgdesc="A pdf to svg converter (mingw-w64)"
+arch=('any')
+url="http://www.cityinthesky.co.uk/opensource/pdf2svg/"
+license=('GPL')
+depends=("${MINGW_PACKAGE_PREFIX}-poppler")
+source=($pkgname-$pkgver.tar.gz::https://github.com/db9052/$_realname/archive/v$pkgver.tar.gz)
```
Comment (Alexpux) [inline mingw-w64-pdf2svg/PKGBUILD]: $_realname-$pkgver.tar.gz please

Comment (yujincheng08) [reply mingw-w64-pdf2svg/PKGBUILD]: Fixed!

File: mingw-w64-pdf2svg/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+url="http://www.cityinthesky.co.uk/opensource/pdf2svg/"
+license=('GPL')
+depends=("${MINGW_PACKAGE_PREFIX}-poppler")
+source=($_realname-$pkgver.tar.gz::https://github.com/db9052/$_realname/archive/v$pkgver.tar.gz)
+sha256sums=('4fb186070b3e7d33a51821e3307dce57300a062570d028feccd4e628d50dea8a')
+
+build() {
+  cd "${srcdir}"/$_realname-$pkgver
+
+  ./configure --prefix=/usr
```
Comment (Alexpux) [inline mingw-w64-pdf2svg/PKGBUILD]: MINGW packages use other prefix, also you must add options --build, --host, --target. See other MINGW packages PKGBUILDs

Comment (yujincheng08) [reply mingw-w64-pdf2svg/PKGBUILD]: Thx a lot and now fixed!

**Pull request merged**